platform/node/index.d.ts

```diff
@@ -125,5 +130,90 @@ declare module '@maplibre/maplibre-gl-native' {
      * Call to permanently dispose the internal map resources, instance can't be used for further render calls
      */
     release: () => void;
+
+    /**
+     * Add source to map's style
+     */
+    addSource: (sourceId: string, source: {}) => void
```

Comment:
```suggestion
    addSource: (sourceId: string, source: object) => void
```

See the [`object` definition](https://www.typescriptlang.org/docs/handbook/2/functions.html#object):

> The special type `object` refers to any value that isn’t a primitive (`string`, `number`, `bigint`, `boolean`, `symbol`, `null`, or `undefined`). This is different from the empty object type `{ }`, and also different from the global type `Object`. It’s very likely you will never use `Object`.

Or would any of those other primitives be desired? Then `any` would be better readable?